jbmc/regression/jbmc-concurrency/get-current-thread/test7.desc

```diff
@@ -0,0 +1,6 @@
+CORE
+A.class
+--function 'A.me:()V' --java-threading --cp `../../../../scripts/format_classpath.sh . ../../../src/java_bytecode/library/core-models.jar` --lazy-methods
```

Comment:
Can you make `test.desc` THOROUGH then? (It takes ~20sec to execute). Or just add `--lazy-methods in `test.desc` instead of adding test7, omitting the non lazy test case.

Reply:
done

Reply:
done